runtime/src/accounts_db.rs

```diff
@@ -3946,17 +3946,19 @@ impl AccountsDb {
         &'a self,
         dead_slots_iter: impl Iterator<Item = &'a Slot> + Clone,
         purged_slot_pubkeys: HashSet<(Slot, Pubkey)>,
-        mut purged_account_slots: Option<&mut AccountSlots>,
+        // If the slot being are not cached, return
+        is_cached: Option<&mut AccountSlots>,
```

Comment:
nits: keep original var name? This paused me a whil because the type isn't bool.